fabbwled-frontend/src/api/character.ts

```diff
@@ -1,40 +1,33 @@
+import { CharacterCreateDto } from "../interfaces/character.ts";
+
 const baseApiURL = "http://localhost:8080/api/player";
-export async function setCharacter(data: CharacterCreateDto): Promise<any> {
-  try {
-    const response = await fetch(baseApiURL, {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-      },
-      body: JSON.stringify(data),
-    });
 
-    if (response.ok) {
-      return await response.json();
-    } else {
-      throw new Error(`API request failed with status: ${response.status}`);
-    }
-  } catch (error: any) {
-    // deliberately used any here
-    throw new Error(`An error occurred: ${error.message}`); // Handle network errors
+export async function setCharacter(data: CharacterCreateDto): Promise<unknown> {
```

Comment:
How do you handle network errors now? (Without a try-catch)

Reply:
They're handled the same way as before, an error gets thrown. Just without the "an error occurred" message

Reply:
Alright 👍